### PR TITLE
Add UNet in apphub

### PR DESCRIPTION
**Pull request by chm123**

### CODE REVIEW

File: fastestimator/op/numpyop/meta/delete.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# limitations under the License.
+# ==============================================================================
+from typing import Any, Dict, Iterable, List, Union
+
+import numpy as np
+
+from fastestimator.op import NumpyOp
+
+
+class Delete(NumpyOp):
```
Comment (TortoiseHam) [inline fastestimator/op/numpyop/meta/delete.py]: not sure this should be a 'meta' op. This is more like univariate. Meta so far refers to Ops which take other Ops as input

File: fastestimator/trace/metric/dice.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        true_key: The key of the ground truth mask
+        pred_key: The key of the prediction values
+        mode: What mode to execute in (None to execute in all modes)
+        output_name: What to call the output from this trace (for example in the logger output)
+    """
+    def __init__(self,
+                 true_key: str,
+                 pred_key: str,
+                 threshold: float = 0.5,
+                 mode: Optional[str] = "eval",
```
Comment (TortoiseHam) [inline fastestimator/trace/metric/dice.py]: could be list of strings also

Comment (TortoiseHam) [reply fastestimator/trace/metric/dice.py]: but also, default should be ("eval", "test") I think

File: apphub/semantic_segmentation/unet_montgomery/unet_montgomery_tf.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        train_data=csv,
+        eval_data=csv.split(0.2),
+        batch_size=batch_size,
+        ops=[
+            ReadImage(inputs="image", parent_path=csv.parent_path, outputs="image", grey_scale=True),
+            ReadImage(inputs="mask_left",
+                      parent_path=csv.parent_path,
+                      outputs="mask_left",
+                      grey_scale=True,
+                      mode='!infer'),
```
Comment (TortoiseHam) [inline apphub/semantic_segmentation/unet_montgomery/unet_montgomery_tf.py]: why would you not read image during inference?

Comment (chm123) [reply apphub/semantic_segmentation/unet_montgomery/unet_montgomery_tf.py]: During inferencing, I don't have masks. Only image is sent to the network.

File: fastestimator/op/numpyop/meta/delete.py (hunk 1 above)
Comment (vbvg2008) [reply fastestimator/op/numpyop/meta/delete.py]: we should extend this definition of meta.    I also have BatchOp that I put in meta folder.

Comment (chm123) [reply fastestimator/op/numpyop/meta/delete.py]: Or create a new category?

File: fastestimator/trace/metric/dice.py (hunk 2 above)
Comment (chm123) [reply fastestimator/trace/metric/dice.py]: fixed

File: apphub/semantic_segmentation/unet_montgomery/unet_montgomery_tf.py (hunk 3 above)
Comment (TortoiseHam) [reply apphub/semantic_segmentation/unet_montgomery/unet_montgomery_tf.py]: oh right

File: fastestimator/op/numpyop/meta/delete.py (hunk 1 above)
Comment (TortoiseHam) [reply fastestimator/op/numpyop/meta/delete.py]: I think those are both examples of univariate. they take one or more keys and act on them each individually

**Pull request closed without merging**